src/cmd-sign

```diff
@@ -71,6 +74,9 @@ def parse_args():
     robosig.add_argument("--gpgkeypath", help="path to directory containing "
                          "public keys to use for signature verification",
                          default="/etc/pki/rpm-gpg")
+    robosig.add_argument("--s3-sigstore", help="bucket and prefix to S3 sigstore")
+    robosig.add_argument("--manifest-list-digest", metavar="ALGO:DIGEST",
```

Comment:
I'm a bit interested to see the update to https://github.com/coreos/fedora-coreos-pipeline/pull/1211 for this one.